scripts/2017/upload.py

```diff
@@ -10,9 +10,11 @@
 # One field is missing in the documentation and in the argument list below: external_url.
 # gci-support says it is coming soon.
 def upload_task(*, name, description, status, max_instances, mentors, tags, is_beginner,
-                categories, time_to_complete_in_days, private_metadata, external_url, do_upload = False):
+                categories, time_to_complete_in_days, private_metadata, external_url=None, do_upload = False):
     task = locals()
     del task['do_upload']
+    if task['external_url'] is None:
+        del task['external_url']
```

Comment:
Add a comment explaining why we're not using `external_url` somewhere around here?

(And update the comment above)